index.js

```diff
@@ -73,7 +74,8 @@ function choo (opts) {
   function render (state, data, prev, name, createSend) {
     if (!_frame) {
       _frame = nanoraf(function (state, prev) {
-        const newTree = _router(state.location.pathname, state, prev)
```

Comment:
`state.location.href` throws this: `Uncaught Error: route ':9966/' did not match`

Would have to fix sheet-router to exclude the port, which is probably a reasonable thing to do anyway.

Yeah I feel like both types of routing shouldn't exist in one app. Does sheet-router need to change to recognize routing with `state.location.query`?